app/controllers/admins/ws_panel_controller.rb

```diff
@@ -233,8 +250,13 @@ def driver_avatar
       driver_data = { 'id' => @driver.id }
       driver_data[:thumb] = @driver.avatar.url(:thumb)
       driver_data[:medium] = @driver.avatar.url(:medium)
-      send_message :update_avatar, driver_data
+      broadcast_message :update_avatar, driver_data
     end
   end
 
+  private
+
+  def last_admin?
+    Admin.where(active: true).length == 1 ? true : false
```

Comment:
I think that it's more relevant to use `.count` instead of `.length` here :construction_worker_man: